src/app/find/mapPage/mapPage.css

```diff
@@ -0,0 +1,86 @@
+@value primaryColor from "../../../colors.css";
+
+.resultsBar{
+	background-color: #ffffff;
+	position: absolute;
+	left: 0;
+	top: 0;
+	right: 0;
+	z-index: 2;
```

Comment:
While I do think separating the styles into separate CSS files is probably a good idea, this is an example of where keeping some styles in the JSX and moving only some of them out into a file is problematic:
The meaning of this zIndex only makes sense when it's compared to the zIndex of the map, which is 1. Anyone looking at this file wouldn't know the map is 1, and anyone looking at the JSX wouldn't know this is 2. This makes it pretty likely that some could change one of these values, without really being aware of the consequences. Also, it makes it harder to get a general sense of the layout by looking at just one place.

Reply:
What should I do?

Reply:
One of the following 2 options (whichever you prefer):
* Put the z-indexes that were moved into this CSS file back into the JSX
* Move the remaining z-indexes that stayed in the JSX into this CSS file as well
As long as they're all in the same place (whether that's the CSS or JSX), anyone touching the code can easily tell how the different elements' indexes relate to each other and what effect changing them would have.

(Personally, I actually find it a bit more convenient when they're in the JSX, because it saves me the need to keep going back and forth between the CSS and JSX to match class names to elements, but it's not a very strong preference.)

Reply:
It looks like there was no z-index in the jsx

Reply:
As mentioned in the call, there is still z-index in the jsx, but since it's a JSX file and not CSS, it's actually `zIndex` and not `z-index`.